README.rst

```diff
@@ -156,3 +156,20 @@ rather than the first one, so multi-maps and simple mappings are binary compatib
 Multi-maps with string keys can also be approximate, meaning the original keys will be discarded and the mapping will
 only work with hashes, making the map much faster and more compact, at the expense of some inaccuracy where the
 returned values could have extra values corresponding to other keys whose hash collide with the one being requested.
+
+Tests
+=====
+
+Running tests can be done locally or on docker, using the script `run-tests.sh`:
```

Comment:
Include the virtualenv steps you had included earlier, that were a good idea.

```
Running tests can be done locally in a virtual env, or alternatively in a docker, using the script `run-tests.sh`:

.. code:: shell

  $> virtualenv venv
  $> . ./venv/bin/activate
  $> ./run-tests.sh
```

Make sure run-tests.sh has execute permission, btw.

Reply:
After that, LGTM